cogboard-webapp/src/components/ZabbixChart/helpers.js

```diff
@@ -0,0 +1,8 @@
+export const calculatePercentageValue = (value, maxValue) => Math.round((100 * value) / maxValue)
+
+export const getNumberOfElements = (array, number) => array.slice(Math.max(array.length - number, 0));
+
+export const convertEpochToDate = (value) => {
+    const convertedEpoch = parseInt(value) + (new Date().getTimezoneOffset() * -1);

```

Comment:
Why this strange `* -1` ?
`const convertedEpoch = parseInt(value) - (new Date().getTimezoneOffset());` isn't this the same